sdk/communication/azure-communication-rooms/pom.xml

```diff
@@ -0,0 +1,178 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under 
+	the MIT License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>com.azure</groupId>
+		<artifactId>azure-client-sdk-parent</artifactId>
+		<version>1.7.0</version> <!-- {x-version-update;com.azure:azure-client-sdk-parent;current} -->
+		<relativePath>../../parents/azure-client-sdk-parent</relativePath>
+	</parent>
+
+	<groupId>com.azure</groupId>
+	<artifactId>azure-communication-rooms</artifactId>
+	<packaging>jar</packaging>
+	<version>1.0.0-alpha.1</version> <!-- {x-version-update;com.azure:azure-communication-rooms;current} -->
+
+	<name>Microsoft Azure library for ACS Room Service</name>
+	<description>
+    This package contains clients and data structures used to create, update, get, delete the Azure Communication Room Service.
+    For this release, see notes - todo
+    Microsoft Azure Communication Rooms quickstart - todo
+  </description>
```

Comment:
nit: fix indentation and also update the formatting of the whole pom file. We generally only have 2 spaces for indentation in pom files.

Reply:
Addressed.